README.md

```diff
@@ -22,6 +22,19 @@ encode changes that we find useful to the source manifests.
 This allows us to readily keep the component configs up-to-date with latest
 versions.
 
+## Purpose
+
+The primary purpose for this project is to maintain manifests that are used in
+other projects to build Kubernetes Operators.  Those projects introduce the
```

Comment:
I don't think our sole purpose is for people to consume these in operators.  I think the purpose is to consume these in any way that they wish to consume them.  For us, obviously, we are after consumption in operators, but I do not believe that is the purpose of this project.